fsw/nos3_defs/cpu1_cfe_es_startup.scr

```diff
@@ -1,48 +1,50 @@
-CFE_APP, /cf/sc.so,                     SC_AppMain,               SC,               50, 16384, 0x0, 0;
+CFE_APP, cf,                        CF_AppMain,               CF,               50, 16384, 0x0, 0;
+CFE_APP, ds,                        DS_AppMain,               DS,               51, 16384, 0x0, 0;
+CFE_APP, fm,                        FM_AppMain,               FM,               52, 16384, 0x0, 0;
+CFE_APP, lc,                        LC_AppMain,               LC,               53, 16384, 0x0, 0;
+CFE_APP, sc,                        SC_AppMain,               SC,               54, 16384, 0x0, 0;
 
-CFE_APP, /cf/arducam.so,                arducam_AppMain,          CAM,              60, 8192,  0x0, 0;
-CFE_APP, /cf/generic_css.so,            GENERIC_CSS_AppMain,      CSS,              60, 8192,  0x0, 0;
-CFE_APP, /cf/generic_eps.so,            GENERIC_EPS_AppMain,      EPS,              60, 8192,  0x0, 0;
-CFE_APP, /cf/generic_fss.so,            GENERIC_FSS_AppMain,      FSS,              60, 8192,  0x0, 0;
-CFE_APP, /cf/generic_imu.so,            GENERIC_IMU_AppMain,      IMU,              60, 8192,  0x0, 0;
-CFE_APP, /cf/generic_mag.so,            GENERIC_MAG_AppMain,      MAG,              60, 8192,  0x0, 0;
-CFE_APP, /cf/generic_radio.so,          GENERIC_RADIO_AppMain,    RADIO,            60, 8192,  0x0, 0;
-CFE_APP, /cf/generic_rw.so,             GENERIC_RW_AppMain,       RW,               60, 8192,  0x0, 0;
-CFE_APP, /cf/generic_torquer.so,        GENERIC_TORQUER_AppMain,  TORQUER,          60, 8192,  0x0, 0;
-CFE_APP, /cf/novatel_oem615.so,         novatel_oem615_AppMain,   NAV,              60, 8192,  0x0, 0;
-CFE_APP, /cf/sample.so,                 SAMPLE_AppMain,           SAMPLE,           60, 8192,  0x0, 0;
-CFE_APP, /cf/generic_adcs.so,           Generic_ADCS_AppMain,     ADCS,             60, 8192,  0x0, 0;
+CFE_APP, arducam,                   arducam_AppMain,          CAM,              60, 8192,  0x0, 0;
+CFE_APP, generic_adcs,              ADCS_AppMain,             ADCS,             61, 8192,  0x0, 0;
+CFE_APP, generic_css,               CSS_AppMain,              CSS,              62, 8192,  0x0, 0;
+CFE_APP, generic_eps,               EPS_AppMain,              EPS,              63, 8192,  0x0, 0;
+CFE_APP, generic_fss,               FSS_AppMain,              FSS,              64, 8192,  0x0, 0;
+CFE_APP, generic_imu,               IMU_AppMain,              IMU,              65, 8192,  0x0, 0;
+CFE_APP, generic_mag,               MAG_AppMain,              MAG,              66, 8192,  0x0, 0;
+CFE_APP, generic_radio,             RADIO_AppMain,            RADIO,            67, 8192,  0x0, 0;
+CFE_APP, generic_rw,                RW_AppMain,               RW,               68, 8192,  0x0, 0;
+CFE_APP, generic_torquer,           TORQUER_AppMain,          TORQUER,          69, 8192,  0x0, 0;
+CFE_APP, novatel_oem615,            NAV_AppMain,              NAV,              70, 8192,  0x0, 0;
```

Comment:
The entry point in novatel_oem615_app.c is not NAV_AppMain so it will not get started.  Change the entry point in novatel_oem615_app.c to this name.

Reply:
Latest commit should resolve this! Good catch

Reply:
This has been resolved.